Release Notes/2017/2017-05-23-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,48 @@
+{{{
+"title": "Cloud Platform - Release Notes: May 23, 2017",
+"date": "5-23-2017",
+"author": "Bob Jackson",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (1)
+
+* __Cloud Application Manager__
+
+- Bulk Import of auto discovered instances
+Cloud Application Manager now allows users to register auto discovered instances in bulk. Full instructions on how to use this new feature are available at the Knowledge Base section on our website.
+
+- Microsoft Azure Resale available for U.K. based customers
+Azure resale to companies based in the United Kingdom is Live! From the Azure Provider, users whose main registered address with CenturyLink is in the United Kingdom, will be permitted to automatically provision new Azure accounts for consolidated billing and platform level support. The capabilities that CenturyLink will Resell will still be the same as defined here https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-azure-capabilities/

```

Comment:
Change link to `../../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-azure-capabilities.md`